submissions/vovan-zt/popup/style.css

```diff
@@ -0,0 +1,190 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+li {
+    list-style-type: none;
+    text-decoration: none;
+}
+
+.contaiter {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: rgba(0, 0, 0, 0.2);
+}
+
+.navigation {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.navigation__list {
+    margin: 1rem;
+    display: grid;
+    grid-template-columns: repeat(5, 50px);
+    grid-template-rows: 40px;
+    gap: 20px;
+    justify-items: center
+}
+
+.navigation__check_input {
+    position: absolute;
+    appearance: none;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+}
+
+.navigation__item  {
+    width: 50px;
+    height: 50px;
+    list-style-type: none;
+    border-radius: 10px;
+    cursor: pointer;
+    padding: 10px; 
+    justify-items: center;
+}
+
+.navigation__item:hover {
+    border-radius:10px;
+    box-shadow: 5px 5px 5px rgba(40, 9, 9, 0.15);
+    background-color:rgba(0, 0, 0, 0.1)
+}
+
+.navigation__check_input:focus ~label {
+    outline-style: solid;
+    outline-width: 0.2rem;
+}
+
+.navigation__image {
+    width: 30px;
+    height: 30px;
+    cursor: pointer;
+}
+
+.modal {
+    width: 400px;
+    position: relative;
+    padding: 10px;
+    top: 10px;
+    right: 180px;
+    display: none;
+   
+}
+
+.modal__list {
+    background-color: rgba(0, 0, 0, 0.1);
+    border-radius: 5px;
+    box-shadow: 5px 5px 10px rgba(0,0,0,0.25);
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-auto-rows: 100px;
+    list-style: none;
+    gap: 30px 30px;
+    padding: 20px;
+    position: relative;
+    overflow: auto;
+    height: 570px;
+}
+
+.modal__list::-webkit-scrollbar {
+    width: 7px;
+    height: 0;
+}
+.modal__list::-webkit-scrollbar-thumb {
+    background: rgb(140, 139, 139);
+    border-radius:5px;
+}
+
+.modal__item {
+    display: flex;
+    flex-direction: column;
+    align-content: space-around;
+    justify-content: start;
+    flex-wrap: wrap;
+    padding: 10px;
+}
+
+.modal__item a {
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)